generators/app/templates/src/app/screens/Auth/screens/Login/index.test.js

```diff
@@ -9,14 +9,18 @@ const VALID_PASSWORD = 'HelloWord1234';
 
 describe('<Login />', () => {
   test('Invalid inputs', async () => {
+    const useDispatchSpy = jest.spyOn(redux, 'useDispatch');
+    const dispatch = jest.fn();
+    useDispatchSpy.mockReturnValue(dispatch);
+
     const { getByText, getAllByText, getByTestId } = render(<Login />);
 
     const submitButton = getByText('Iniciar sesión');
     const emailInput = getByTestId('Email');
 
     fireEvent.press(submitButton);
-    await waitFor(() => getAllByText('Este campo es obligatorio'));
-    expect(getAllByText('Este campo es obligatorio')).toHaveLength(2);
+
+    await waitFor(() => expect(getAllByText('Este campo es obligatorio')).toHaveLength(2));
```

Comment:
shouldn't this be a i18n response?

Reply:
I agree that we should use i18n in the tests. But I'd tackle that in another PR, when we add additional tests to the bootstrap